app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -155,6 +161,7 @@ public void run() {
                 } catch (JSONException e) {
                     backoff(e);
                 }
```

Comment:
@timabbott After changing from Apache HTTP to okHttpClient asyncGetEvents for the queue works a lot faster and hence it floods the server, so currently I gave a 1 sec pause after every heartbeat receives, will this be the right interval time or increasing this will also do fine? 
The current app seems fine though in receiving new messages!